### ISSUE TITLE
unit test failures against C* 1.1

### ISSUE THREAD

**Issue by datastaxqa** [driftx/Telephus#23]
``````
test.test_cassandraclient
  CassandraClientTest
    test_bad_params ...                                               [SKIPPED]
    test_batch_insert_get_slice_and_count ...                              [OK]
    test_batch_mutate_and_remove ...                                       [OK]
    test_batch_mutate_with_deletion ...                                    [OK]
    test_column_family_manipulation ...                                  [FAIL]
    test_counter_add ...                                              [SKIPPED]
    test_counter_remove ...                                           [SKIPPED]
    test_describes ...                                                     [OK]
    test_errback ...                                                       [OK]
    test_indexed_slices ...                                                [OK]
    test_insert_get ...                                                    [OK]
    test_keyspace_manipulation ...                                         [OK]
    test_multiget_slice_remove ...                                         [OK]
    test_range_slices ...                                                  [OK]
    test_ttls ...                                                          [OK]
  ManagedCassandraClientFactoryTest
    test_api_match ...                                                     [OK]
    test_initial_connection_failure ...                                    [OK]

===============================================================================
[SKIPPED]
Disabled pending further investigation...

test.test_cassandraclient.CassandraClientTest.test_bad_params
===============================================================================
[SKIPPED]
Counters are not supported in 0.7

test.test_cassandraclient.CassandraClientTest.test_counter_add
test.test_cassandraclient.CassandraClientTest.test_counter_remove
===============================================================================
[FAIL]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/twisted/internet/defer.py", line 1039, in _inlineCallbacks
    result = g.send(result)
  File "/var/lib/jenkins/jobs/Telephus/workspace/test/test_cassandraclient.py", line 366, in test_column_family_manipulation
    self.assertEqual(cfdef, cfdef2)
  File "/usr/local/lib/python2.7/site-packages/twisted/trial/unittest.py", line 270, in assertEqual
    % (msg, pformat(first), pformat(second)))
twisted.trial.unittest.FailTest: not equal:
a = CfDef(comment='foo', key_validation_class=None, min_compaction_threshold=5, key_cache_save_period_in_seconds=3600, gc_grace_seconds=86400, default_validation_class='org.apache.cassandra.db.marshal.BytesType', max_compaction_threshold=31, read_repair_chance=1.0, key_cache_size=200000.0, id=1016, replicate_on_write=None, subcomparator_type=None, merge_shards_chance=None, row_cache_provider=None, row_cache_save_period_in_seconds=0, column_type='Standard', memtable_throughput_in_mb=None, memtable_flush_after_mins=None, column_metadata=[], key_alias=None, name='TransientCF', keyspace='TelephusTests', memtable_operations_in_millions=None, comparator_type='org.apache.cassandra.db.marshal.BytesType', row_cache_size=0.0)
b = CfDef(comment='foo', key_validation_class=None, min_compaction_threshold=5, key_cache_save_period_in_seconds=None, gc_grace_seconds=86400, default_validation_class='org.apache.cassandra.db.marshal.BytesType', max_compaction_threshold=31, read_repair_chance=1.0, key_cache_size=200000, id=1016, replicate_on_write=None, subcomparator_type=None, merge_shards_chance=None, row_cache_provider=None, row_cache_save_period_in_seconds=None, column_type='Standard', memtable_throughput_in_mb=None, memtable_flush_after_mins=None, column_metadata=[], key_alias=None, name='TransientCF', keyspace='TelephusTests', memtable_operations_in_millions=None, comparator_type='org.apache.cassandra.db.marshal.BytesType', row_cache_size=0)


test.test_cassandraclient.CassandraClientTest.test_column_family_manipulation
-------------------------------------------------------------------------------
Ran 17 tests in 12.956s

FAILED (skips=3, failures=1, successes=13)```
``````

**Comment by thepaul** [comment]
Pushed a fix for testing against 1.1 to the unstable branch.

**Comment by datastaxqa** [comment]
Thanks, thepaul.  Fix confirmed.
